apps/render_cf.py

```diff
@@ -185,6 +192,7 @@ def render_costs(job_types: dict, cost_profile: str) -> None:
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument('-j', '--job-spec-files', required=True, nargs='+', type=Path)
+    parser.add_argument('-e', '--compute-environment-files', nargs='+', type=Path)
```

Comment:
Let's at least simplify the `--compute-environment-files` option so that it takes a single string (and rename it to `--compute-environment-file`), to make it clear that there's only one compute environment file.